src/transport_performance/urban_centres/raster_uc.py

```diff
@@ -325,25 +336,25 @@ def _cluster_cells(
 
     def _check_cluster_pop(
         self,
-        band: np.ndarray,
+        band: Union[np.ndarray, ma.core.MaskedArray],
```

Comment:
I was avoiding using `ma.core.MaskedArray`, and was using `np.ndarray` instead for every instance. I was doing this because `ma.core.MaskedArray` is a child class of `np.ndarray`:

```
>>> from numpy.ma import MaskedArray
>>> import numpy as np
>>> m = MaskedArray()
>>> isinstance(m, np.ndarray)

True
```

Happy to specifically refer to class `ma.core.MaskedArray` if useful.

Reply:
Hi @SergioRec , I'll leave this decision to you. I was simply changing the format of the Union statement in the docstring. Prior to my change, it [looked like this](https://github.com/datasciencecampus/transport-network-performance/blob/c118cbcc667c71f7187b7a603f24677de7bb6e0d/src/transport_performance/urban_centres/raster_uc.py#L340):

> band : np.ndarray or ma.core.MaskedArray

Reply:
Thanks @r-leyshon , this was an oversight on my part as I thought I had replaced all instances by `np.ndarray`. Corrected now.

Reply:
Just seen @SergioRec , thanks! No worries, I'm experimenting with hotfixing some of these small changes rather than leaving review comments as potentially a quicker way of tidying things. Though in me making the changes, I seem to be losing the context of the change for the author. Something for me to consider when reviewing.